user/TASK/chassis_task/chassis_task.c

```diff
@@ -199,12 +211,19 @@ char pid_out[64];
  * @retval None
  */
 static void increment_PID(Chassis_t *chassis_pid){
-	//translation
+    
+    for(int i = 0; i < 4; i++){
+        chassis_pid->motor[i].current_out = chassis_pid->motor[i].speed_set;
+    }
+	
+    //translation
     //rotation
 	fp32 front_right;
     fp32 back_right;
     fp32 front_left;
     fp32 back_left;
+    
+    // TODO: eliminate in place declaration & cast of floats 
```

Comment:
I don't think we should push minor TODO's like this. Also,
- "eliminate in place declaration" - what is wrong with the in place declarations? What would you recommend replaces this?
- "cast of floats" - should be made explicit - it currently takes some digging to find out where the cast is. If we do end up casting, I think we should cast straight out of PID_Calc() instead of at the +- .current_out line - it is more clear if you cast earlier